data/101/748/149/101748149.geojson

```diff
@@ -268,6 +271,9 @@
         "Pernov",
         "Pyaonu"
     ],
+    "name:unk_x_variant":[
+        "P\u0113rnava"
```

Comment:
Where is this unknown name variant coming from?

Reply:
https://github.com/whosonfirst-data/whosonfirst-data-admin-ee/pull/11#discussion_r463332361, same applies here